app/controllers/helper/ShapefilesCreatorHelper.java

```diff
@@ -0,0 +1,443 @@
+                                + // line geometry
+                                "name:String,"
+                                + // <- Neighborhood Name
+                                "regionId:Integer,"
+                                + // <- Neighborhood Id
+                                "coverage:Double,"
+                                + // coverage score
+                                "score:Double," 
+                                + // obstacle score
+                                "sigRamp:Double,"
+                                + // curb ramp significance score
+                                "sigNoRamp:Double,"
+                                + // no Curb ramp significance score
+                                "sigObs:Double,"
+                                + // obstacle significance score
+                                "sigSurfce:Double," 
+                                + // Surface problem significance score
+                                "nRamp:Double,"
+                                + // curb ramp feature score
+                                "nNoRamp:Double,"
+                                + // no Curb ramp feature score
+                                "nObs:Double,"
+                                + // obstacle feature score
+                                "nSurfce:Double" // Surface problem feature score
+
+                );
+
+
+
+        /*
+         * A list to collect features as we create them.
+         */
+        List<SimpleFeature> features = new ArrayList<>();
+
+        /*
+         * GeometryFactory will be used to create the geometry attribute of each feature,
+         * using a Point object for the location.
+         */
+        GeometryFactory geometryFactory = JTSFactoryFinder.getGeometryFactory();
+
+        SimpleFeatureBuilder featureBuilder = new SimpleFeatureBuilder(TYPE);
+
+        for(NeighborhoodAttributeSignificance n : neighborhoods){
+            featureBuilder.add(geometryFactory.createPolygon(n.geometry()));
+            featureBuilder.add(n.name());
+            featureBuilder.add(n.regionID());
+            featureBuilder.add(n.coverage());
+            featureBuilder.add(n.score());
+            featureBuilder.add(n.significanceScores()[0]);
+            featureBuilder.add(n.significanceScores()[1]);
+            featureBuilder.add(n.significanceScores()[2]);
+            featureBuilder.add(n.significanceScores()[3]);
+            featureBuilder.add(n.attributeScores()[0]);
+            featureBuilder.add(n.attributeScores()[1]);
+            featureBuilder.add(n.attributeScores()[2]);
+            featureBuilder.add(n.attributeScores()[3]);
+
+            SimpleFeature feature = featureBuilder.buildFeature(null);
+            features.add(feature);
+        }
+        createGeneralShapeFile(outputFile, TYPE, features);
+
+    }
+
+
+    public static File zipShapeFiles(String name, String[] files) throws IOException{
```

Comment:
Could you add comment(s) for this method? Also go through it to check for small style things like having a space before open curly braces.

Reply:
Done!